common/history/countries/0_dvg_subsaharan_africa.txt

```diff
@@ -76,10 +76,33 @@
 		
 		set_ruling_interest_groups = {
 			ig_armed_forces
+			ig_landowners
 		}
 		ig:ig_devout = { 
-			set_interest_group_name = ig_evangelical_church
+			set_interest_group_name = ig_dutch_reformed_church
+		}
+		ig:ig_armed_forces= { 
```

Comment:
If these interest group name and ideology are culture specific, you can move it to the interest group on_enable = {.
This is new feature since 1.4